ui/app/AppLayouts/Profile/panels/ProfileShowcasePanel.qml

```diff
@@ -315,32 +406,49 @@ DoubleFlickableWithFolding {
                 property var dragParentData: root
                 property int visualIndexData: showcaseDelegateRoot.index
                 property var dragKeysData: showcaseDelegateRoot.isHiddenShowcaseItem ?
-                                           d.dragHiddenItemKey : d.dragShowcaseItemKey
+                                               d.dragHiddenItemKey : d.dragShowcaseItemKey
 
                 width: parent.width
                 sourceComponent: root.delegate
                 onItemChanged: {
                     if (item) {
-                        item.showcaseVisibilityRequested.connect((toVisibility) => root.setVisibilityRequested(showcaseDelegateRoot.model.showcaseKey, toVisibility))
+                        item.showcaseVisibilityRequested.connect((toVisibility) => d.setVisibilityInternalRequested(showcaseDelegateRoot.model.showcaseKey, toVisibility))
                     }
                 }
             }
 
             Binding {
-                 when: showcaseDelegateRoot.isHiddenShowcaseItem ? d.isAnyShowcaseDragActive : d.isAnyHiddenDragActive
-                 target: showcaseDraggableDelegateLoader.item
-                 property: "blurState"
-                 value: true
-                 restoreMode: Binding.RestoreBindingOrValue
-             }
-
-             Binding {
+                when: showcaseDelegateRoot.isHiddenShowcaseItem ? d.isAnyShowcaseDragActive : (d.isAnyHiddenDragActive ||
+                                                                                               (d.isAnyHiddenDragActive && d.limitReached))
+                target: showcaseDraggableDelegateLoader.item
+                property: "blurState"
+                value: true
+                restoreMode: Binding.RestoreBindingOrValue
+            }
+
+            Binding {
                 when: showcaseShadow.visible
                 target: d
                 property: showcaseDelegateRoot.isHiddenShowcaseItem ? "isAnyHiddenDragActive" : "isAnyShowcaseDragActive"
                 value: true
                 restoreMode: Binding.RestoreBindingOrValue
-             }
+            }
+
+            Binding {
+                when: showcaseDelegateRoot.isHiddenShowcaseItem && d.limitReached
+                target: showcaseDraggableDelegateLoader.item
+                property: "contextMenuEnabled"
+                value: false
+                restoreMode: Binding.RestoreBindingOrValue
+            }
+
+            Binding {
+                when: showcaseDelegateRoot.isHiddenShowcaseItem  && d.limitReached
+                target: showcaseDraggableDelegateLoader.item
+                property: "tooltipTextWhenContextMenuDisabled"
+                value: qsTr("Showcase limit of %1 reached. <br>Remove item from showcase to add more.").arg(root.showcaseLimit)
```

Comment:
`Remove item from showcase to add more.` Is this proper English? 🤔 Sounds weird to me!

Reply:
I'm not the best to anylise that :sweat_smile:.. Here it is the [figma](https://www.figma.com/file/ibJOTPlNtIxESwS96vJb06/%F0%9F%91%A4-Profile-%7C-Desktop?type=design&node-id=5181%3A265342&mode=design&t=izNswbj2OVZB4l9C-1) but let's confirm with @benjthayer

Reply:
@noeliaSD @alexjba yeah the English is ok - it's not how you would say it in conversation but for the UI I think it is fine - it's just an abbreviated form of `Remove an item from the showcase in order to add more items`

Open to suggestions though!

Reply:
Thanks @benjthayer!